src/site/content/en/index.njk

```diff
@@ -128,11 +128,19 @@ eleventyComputed:
     </div>
     <div class="w-grid w-pb--std">
       <div class="w-grid__columns w-grid__columns--featured" role="list">
-        {% for post in collections.recentBlogPosts %}
-          {% if loop.first %}
-            {% PostCard post, true %}
-          {% else %}
-            {% PostCard post %}
+        {% set counter = 0 %}
+        {% for post in collections.blogPosts %}
```

Comment:
Not saying that it's computationally intensive to iterate over all of the objects here, I doubt it is, but I think this could be made cleaner and easier to read...

The logic could be moved into an 11ty data file like so...

```JavaScript
module.exports = {
    eleventyComputed: {
        recentPosts: (data) => {
            const recent = [];
            for (const item of data.collections.blogPosts) {
                if (item.data.hero || item.data.thumbnail && item.data.lang === data.lang) {
                    recent.push(item);
                    if (recent.length === 3) { break; }
                }
            }
            return recent;
        }
      }
};
```

And then in the njk file the for loop would just be:

```njk
{% for post in recentPosts %}
    {% PostCard post, loop.first %}
{% endfor %}
```